ModuleSystem/module_scripts_common_warp.py

```diff
@@ -0,0 +1,764 @@
+from header_common import *
+from header_operations import *
+from module_constants import *
+from header_parties import *
+from header_skills import *
+from header_mission_templates import *
+from header_items import *
+from header_triggers import *
+from header_terrain_types import *
+from header_music import *
+from ID_animations import *
+from ID_troops import *
+from ID_factions import *
+#from module_troops import *
+
+from module_info import wb_compile_switch as is_a_wb_script
+
+# Credits to Autolykos, thread https://forums.taleworlds.com/index.php?topic=335511.0
```

Comment:
@ehsn I wasn't aware of this. Haven't really been up-to-date with M&B modding since 2014. This is pretty inventive. Good find!